.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
- [ ] Congratulations, you have perfectly ignored the node_modules file, but one thing is missing, the `test.md` file, I dare to say maybe you did not create one, since it is not in the `.gitignore` file, at least we should be able to see it in the repo, but it is not anywhere... then reading the [requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), please create the `test.md` file, add the said text in it and finally add it to the `.gitignore`, after that, you will be good to go resubmit your project